CONTRIBUTING.md

```diff
@@ -1,15 +1,143 @@
 # Contributing to kubernetes/test-infra
 
-Make a [pull request](https://help.github.com/articles/about-pull-requests/) (hereafter PR). 
-In the PR body, feel free to add an area label if appropriate by saying `/area <AREA>`. 
-The list of labels is [here](https://github.com/kubernetes/test-infra/labels). 
-Also feel free to suggest a reviewer with `/assign @theirname`.
-
-Once your reviewer is happy, they will say `/lgtm` which will apply the 
-`lgtm` label, and will apply the `approved` label if they are an 
-[owner](/OWNERS).
-The `approved` label will also automatically be applied to PRs opened by an 
-OWNER. If neither you nor your reviewer is an owner, please `/assign` someone
- who is.
-Your PR will be automatically merged once it has the `lgtm` and `approved` 
-labels, does not have any `do-not-merge/*` labels, and all tests are passing.
+Welcome! If you haven't seen them already, the Kubernetes project has:
+
+- A [Contributor Guide][contrib-guide] - some [kubernetes/kubernetes]-specific content, but lots of info for the entire project
+- A [Contributor Cheat Sheet][contrib-cheatsheet] - lots of resources and handy links
+
+## Contact
+
+This repo is owned by [SIG Testing][sig-testing], and the people most directly
+involved are listed in [/OWNERS]. We are most responsive on our slack channel
+
+- Community Site: [kubernetes/community/sig-testing][sig-testing]
+- Slack: [#sig-testing]
+- Mailing List: [kubernetes-sig-testing@]
+
+We also have two other slack channels:
+
+- [#testing-ops]: for [prow.k8s.io] and other test-infra ops. Ping `@test-infra-oncall` or check [go.k8s.io/oncall]
+- [#prow]: for discussions related to the use or development of [`prow`](/prow)
+
+## Workflow
+
+In general we use the same pull request (PR) and review workflow used by the
+rest of the Kubernetes project. This means the [use of a CLA][cla], [code
+review by reviewers and approvers listed in OWNERS files][owners], and tests
+that automatically exercise code or enforce conventions. Many PRs will result
+in changes automatically being deployed.
+
+Some conventions that are specific to this repo:
+
+- For large code changes, please write a design doc or [KEP] and get signoff
+  from SIG Testing before trying to land code. If you're not sure what a KEP
+  should look like, [kuberenetes/enhancements/keps/sig-testing] has some
+  examples. If you're not sure what "large" means, [come ask us](#contact)
+- We find it polite to apply the `do-not-merge/hold` label via the `/hold`
+  command when in doubt about whether/when a PR should merge. If we don't
+  explain why we're adding a `/hold`, this is usually because the PR will
+  cause changes to be deployed on merge, and we want the PR author to
```

Comment:
is this generally true currently when the reason isn't explained, or is it say, the oncall that is intended to `/hold cancel` it?

Reply:
Changed wording to case-by-case, I'm open to suggestions on how to delineate.  Changing jobs, enabling plugins, etc. I would all expect to be self-service.  Bumping prow maybe not so much.